packages/react/src/components/ComposedModal/next/ModalFooter.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../Button';
+import ButtonSet from '../../ButtonSet';
+import classNames from 'classnames';
+import { usePrefix } from '../../../internal/usePrefix';
+
+export const ModalFooter = React.forwardRef(function ModalFooter(
+  {
+    className,
+    primaryClassName,
+    secondaryButtons,
+    secondaryClassName,
+    secondaryButtonText,
+    primaryButtonText,
+    primaryButtonDisabled,
+    closeModal, // eslint-disable-line
+    onRequestClose, // eslint-disable-line
+    onRequestSubmit, // eslint-disable-line
```

Comment:
```suggestion
    closeModal,
    onRequestClose,
    onRequestSubmit,
```

I believe these should be okay now, or is there a specific reason for the disable? 👀